packages/appium/lib/main.js

```diff
@@ -384,10 +384,10 @@ export {finalizeSchema, getSchema, validate} from './schema/schema';
 export {main, init, resolveAppiumHome};
 
 /**
- * @typedef {import('appium/types').DriverType} DriverType
- * @typedef {import('appium/types').PluginType} PluginType
- * @typedef {import('appium/types').DriverClass} DriverClass
- * @typedef {import('appium/types').PluginClass} PluginClass
+ * @typedef {import('@appium/types').DriverType} DriverType
+ * @typedef {import('@appium/types').PluginType} PluginType
+ * @typedef {import('@appium/types').DriverClass} DriverClass
+ * @typedef {import('@appium/types').PluginClass} PluginClass
  * @typedef {import('appium/types').WithServerSubcommand} WithServerSubcommand
```

Comment:
is there a reason you didn't update this type reference to `@appium/types`?

Reply:
just putting it here for discoverability: `appium/types` is not `@appium/types`. the former is an alias to the `types` dir in the `appium` package; the latter is the `@appium/types` package itself.